yesod-test/yesod-test.cabal

```diff
@@ -1,5 +1,5 @@
 name:               yesod-test
-version:            1.5.0.1
+version:            1.6.0.0
```

Comment:
What's the reason for the major version bump? It looks like this only changes internal functions.